backoffice/backoffice/workflows/api/utils.py

```diff
@@ -1,11 +1,22 @@
-from backoffice.workflows.api.serializers import DecisionSerializer
+from backoffice.workflows.api import serializers
```

Comment:
since it's one, I wouldn't import the whole module

Reply:
If I only import one, Its a circular import otherwise, therefore the change